bot/exts/info/information.py

```diff
@@ -524,13 +524,17 @@ async def json(self, ctx: Context, message: Message) -> None:
         await self.send_raw_content(ctx, message, json=True)
 
     @command(aliases=("rule",))
-    async def rules(self, ctx: Context, *args: Optional[str]) -> Optional[Set[int]]:
+    async def rules(self, ctx: Context, args: Optional[str]) -> Optional[Set[int]]:
         """
         Provides a link to all rules or, if specified, displays specific rule(s).
 
         It accepts either rule numbers or particular keywords that map to a particular rule.
         Rule numbers and keywords can be sent in any order.
         """
+        # Temporary fix for discord.py greedy string quote conversion bug
+        if not args:
+            args = ("",)
```

Comment:
Shouldn't this be an empty tuple?

Reply:
```py
if args is None:
    args = ()
else:
    args = (args,)
```

Second case is important too since the rest of the code expects `args` to be an iterable of strings.

Reply:
> Shouldn't this be an empty tuple?

Fixed in latest push



> I think it should be
> 
> ```python
> if args is None:
>     args = ()
> else:
>     args = (args,)
> ```
> 
> Second case is important too since the rest of the code expects `args` to be an iterable of strings.

Have changed args to be derived from ctx.message instead